lib/benchpark/runtime.py

```diff
@@ -68,29 +68,43 @@ def __init__(self, dest, upstream=None):
         self.dest = pathlib.Path(dest)
         self.upstream = upstream
 
-        self.ramble_location, self.spack_location = (
+        self.ramble_location, self.spack_location, self.pkgs_location = (
             self.dest / "ramble",
-            self.dest / "spack",
+            self.dest / "spk",
```

Comment:
We should do https://github.com/LLNL/benchpark/pull/1009 to avoid needing to change the spack name